package.json

```diff
@@ -36,9 +36,10 @@
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
     "@types/sinon": "^9.0.8",
+    "@types/supertest": "^2.0.10",
     "@types/winston": "^2.4.4",
     "babel-loader": "^8.1.0",
-    "body-parser": "^1.19.0",
```

Comment:
**Annotation:** Please refrain from adding unused packages to `package.json`. Second warning.